### ISSUE TITLE
records: rendering issue 

### ISSUE THREAD

**Issue by lnielsen** [zenodo/zenodo#264]
https://zenodo.org/record/10096

**Comment by bouzlibop** [comment]
record has multiple dois (`type(record.doi) == list`)

```python
Traceback (most recent call last):
  File "/home/bouzlibop/.virtualenvs/zenodo/src/invenio/invenio/modules/formatter/engine.py", line 504, in format_record_1st_pass
    **kwargs)
  File "/home/bouzlibop/.virtualenvs/zenodo/src/invenio/invenio/modules/formatter/engine.py", line 387, in format_record
    template, bfo, verbose=verbose, extra_context=kwargs)
  File "/home/bouzlibop/.virtualenvs/zenodo/src/invenio/invenio/modules/formatter/engine.py", line 701, in format_with_format_template
    bfo=bfo, **ctx).encode('utf-8')
  File "/home/bouzlibop/.virtualenvs/zenodo/src/invenio/invenio/ext/template/__init__.py", line 61, in render_template_to_string
    return template.render(context)
  File "/home/bouzlibop/.virtualenvs/zenodo/local/lib/python2.7/site-packages/jinja2/environment.py", line 969, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/bouzlibop/.virtualenvs/zenodo/local/lib/python2.7/site-packages/jinja2/environment.py", line 742, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/bouzlibop/.virtualenvs/zenodo/src/zenodo/zenodo/base/templates/format/record/Default_HTML_info.tpl", line 21, in top-level template code
    <dd>{% if record.doi|is_local_doi %}{{doi_badge(record.doi)}}{% else %}{{ record.doi|doi_link }}{% endif %}</dd>
  File "/home/bouzlibop/.virtualenvs/zenodo/src/zenodo/zenodo/base/views.py", line 203, in is_local_doi
    return value.startswith(cfg['CFG_DATACITE_DOI_PREFIX']) or \
AttributeError: 'list' object has no attribute 'startswith'

```

**Comment by bouzlibop** [comment]
Today I encountered following problem:

When I create new record, then edit it and add some new doi as `is alternate identifier` it results in this issue's error. It is because after doing that `<record>.get('doi')` returns list of dois instead of unique doi.

**Comment by lnielsen** [comment]
Fixed in 7697a0423f16b652c02f25739edbdeb7fc93f207
